### PR TITLE
instcountci: Adds panicspill test from steamwebhelper

### PR DESCRIPTION
**Pull request by Sonicadvance1**
This block of code causes our RA to panic spill and it gets hit on steamwebhelper.
We should be able to spill without panic on this code at some point.

#3388 open to track the panic spill in this code.

### CODE REVIEW

**Comment by alyssarosenzweig** [comment]
This isn't really a unit test. Any chance we could get this in InstCountCI instead? :smile_cat:

**Comment by Sonicadvance1** [comment]
> This isn't really a unit test. Any chance we could get this in InstCountCI instead? 😸

Moved to instcountci

**Comment by alyssarosenzweig** [comment]
Oof that's a lot of asm. but yeah thanks :+1:

**Review by alyssarosenzweig — APPROVED** [review]

File: unittests/InstructionCountCI/FEXOpt/libnss.json [hunk 1]
```diff
@@ -0,0 +1,4532 @@
+{
+  "Features": {
+    "Bitness": 64,
+    "EnabledHostFeatures": [
+      "CRYPTO"
```
Comment (alyssarosenzweig) [inline unittests/InstructionCountCI/FEXOpt/libnss.json:5]: explicitly enable flagm? rmif is used

Comment (Sonicadvance1) [reply unittests/InstructionCountCI/FEXOpt/libnss.json]: Explicitly enabled.

**Review by lioncash — APPROVED** [review]

**Pull request merged**